acceptance/apps_test.go

```diff
@@ -227,4 +228,60 @@ var _ = Describe("Apps", func() {
 			}, "1m").Should(MatchRegexp(`Status .*\|.* 1\/1`))
 		})
 	})
+
+	Describe("logs", func() {
+		var (
+			appName, route string
+			logLength      int
+		)
+
+		BeforeEach(func() {
+			appName = newAppName()
+			out := makeApp(appName, 1, true)
+			routeRegexp := regexp.MustCompile(`https:\/\/.*omg.howdoi.website`)
+			route = string(routeRegexp.Find([]byte(out)))
+
+			out, err := Epinio("app logs "+appName, "")
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			podNames := getPodNames(appName, org, 1)
+			for _, podName := range podNames {
+				Expect(out).To(ContainSubstring(podName))
+			}
+			logs := strings.Split(out, "\n")
+			logLength = len(logs)
+		})
+
+		AfterEach(func() {
+			deleteApp(appName)
+		})
+
+		It("follows logs", func() {
+			p, err := GetKCommand("app logs --follow "+appName, "")
+			Expect(err).NotTo(HaveOccurred())
+
+			go p.Run()
```

Comment:
this is an external command, I wonder if it stops after tests are done. Maybe we should Kill that process manually.

Reply:
Why should it stop ? AFAIK the `--follow` tells it to not stop. On the other hand, if the application it is following is deleted, then it will stop ? ... Checked, no, it does not stop even then.

So, right now `log --follow` can only be stopped by a manual Kill.

Reply:
I added the `defer` above to manually kill it but when I simply run `p.Process.Kill()` I got an error because `p.Process` was nil. Maybe the library we use doesn't properly expose the underlying process for killing?

Reply:
p.Process it not nil after calling `p.Run()` so the handling we did is fine. Also fixed the order of things here: https://github.com/epinio/epinio/commit/bdffb6edbf4b293e8fc1dff39381b52bac9fcd40